packages/components/src/alert/docs/Alert.stories.mdx

```diff
@@ -20,24 +20,24 @@ import { Heading, InnerHeading } from "@components/typography";
     githubPath="/packages/components/src/alert/src"
 />
 
-## Accessibility
+## Guidelines
 
-An alert comes with the `alertdialog` role and should be used to display a confirmation message, an alert, an error, or a warning.
+### When to use
 
-It requires:
-- The focus to be trapped inside the alert.
-- A control focused when the dialog appears, provided by the `primaryButtonLabel` prop.
-- A label, provided by the `<Heading>`.
-- A description, provided by the `<Content>`.
+- To display a confirmation message, an error, or a warning.
 
-Label and description can be overwritten with the use of `aria-label` and `aria-describedby` props on the alert.
+### Accessibility
 
-### Focus
+- `aria-label` and `aria-describedby` defaults to the alert `<Heading>` and `<Content>` values.
+- You can overwrite them by setting `aria-label` and `aria-describedby` props on the alert.
 
-If the alert calls for an action, make sure to set the focus correctly to avoid any unwanted action.
-To do so, you can use the `autoFocusButton` prop on the alert to specify which control should have the initial focus.
+[aria-label](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-label), [aria-describedby](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-describedby)
```

Comment:
Un truc qui pourrait être testé ici, c'est le [Footnotes]([aria-label](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-label)).

```
- `aria-label`[^1] and `aria-describedby`[^2] defaults to the alert `<Heading>` and `<Content>` values.
- You can overwrite them by setting `aria-label` and `aria-describedby` props on the alert.

[^1]: [aria-label](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-label)
[^2]: [aria-describedby](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-describedby)
```

donnerai

- `aria-label`[^1] and `aria-describedby`[^2] defaults to the alert `<Heading>` and `<Content>` values.
- You can overwrite them by setting `aria-label` and `aria-describedby` props on the alert.

[^1]: [aria-label](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-label) (mdn web docs)
[^2]: [aria-describedby](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-describedby) (mdn web docs)

Reply:
You could also decide to make all the links footnotes and have the footnote at the very very bottom of the page like:

Source:
```
## Guidelines

### When to use

- To display a confirmation message, an error, or a warning. This is a link[^when] of the first section

### Accessibility

- `aria-label`[^a11y] and `aria-describedby`[^a11y] defaults to the alert `<Heading>` and `<Content>` values.
- You can overwrite them by setting `aria-label` and `aria-describedby` props on the alert.

## Usage

Use it like you want to use it!

[^when]: [my random link](*)
[^a11y]: [aria-label](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-label), [aria-describedby](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-describedby)
```

## Guidelines

### When to use

- To display a confirmation message, an error, or a warning. This is a link[^when] of the first section

### Accessibility

- `aria-label`[^a11y] and `aria-describedby`[^a11y] defaults to the alert `<Heading>` and `<Content>` values.
- You can overwrite them by setting `aria-label` and `aria-describedby` props on the alert.

## Usage

Use it like you want to use it!

[^when]: [my random link](*)
[^a11y]: [aria-label](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-label), [aria-describedby](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-describedby)

Reply:
With the fact that linking in a `code` tag would you still suggest footnotes?

![image](https://user-images.githubusercontent.com/361632/166508534-416fbb2b-c754-46cf-af40-91c1a0088357.png)

Reply:
I think that that in-code link is a good option